### PR TITLE
Create tweets

### PR DESCRIPTION
**Pull request by Ru-T**
## What Changed?
- Created Tweets table
- Allow user to create Tweet
- Display a user's Tweets on their profile and others profiles
## Why?
- To allow users to be able to Tweet and view Tweets, as an integral part of Twitter clone

### CODE REVIEW

File: app/views/tweets/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+      <ul>
+      <% @tweet.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <%= f.hidden_field :user_id %>
```
Comment (BrandonMathis) [inline app/views/tweets/new.html.erb]: Interesting. Was the original idea to hide the ID of the current user here so you didn't need to write [this](https://github.com/smashingboxes/Ruti-Twitter/pull/3/files#diff-eb437675fc47a98939a642af298c1a19R11) code?

That would be clever but a smart user would change the value of this field and make a tweet looked like it came from any user! That would be bad :(

File: spec/models/tweet_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Tweet, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (BrandonMathis) [inline spec/models/tweet_spec.rb:4]: You could either delete this file or remove the `pending`

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (BrandonMathis) [inline spec/models/user_spec.rb]: You could either delete this file or remove the `pending`

**Comment by BrandonMathis** [comment]
You will want to add .DS_Store to you gitignore

**Comment by Ru-T** [comment]
@BrandonMathis fixed all the things you commented on. Can I merge? Thanks!

**Pull request merged**